### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`mixin` -> `@apply`
`x-if` -> `dom-if`
`version++`
`polymer++`

/cc @morethanreal

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-    "paper-styles": "PolymerElements/paper-styles#^0.8.0"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1",
+    "iron-input": "PolymerElements/iron-input#^0.9.0",
+    "paper-styles": "PolymerElements/paper-styles#^0.9.0"
   },
   "devDependencies": {
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.6",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.8.2",
-    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.9.6",
```
Comment (cdata) [inline bower.json]: There is no `0.9.6`

File: bower.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.9.2",
```
Comment (cdata) [inline bower.json]: There is no `0.9.2`

**Comment by cdata** [comment]
https://github.com/PolymerElements/paper-input/blob/c0eeac6d5c53b23500fea316d61bc26e0f12cdaa/paper-input-container.html#L18-L25

The above lines could probably be converted to inline defaults. I confirmed with @sorvell that you can use a variable as a default for another variable. Example:

```css
.foo {
  color: var(--foo-color, var(--default-color));
}
```

File: paper-input.html [hunk 3]
```diff
@@ -24,17 +24,17 @@
 
     <paper-input-container no-label-float$="[[noLabelFloat]]" auto-validate$="[[autoValidate]]" disabled$="[[disabled]]">
 
-      <template is="x-if" if="[[label]]">
+      <template is="dom-if" if="[[label]]">
         <label>[[parent.label]]</label>
```
Comment (cdata) [inline paper-input.html]: `parent` is deprecated.

File: paper-input.html [hunk 4]
```diff
@@ -24,17 +24,17 @@
...
       </template>
 
       <input is="iron-input" id="input" bind-value="{{value}}" prevent-invalid-input="[[preventInvalidInput]]" type$="[[type]]" pattern$="[[pattern]]" maxlength$="[[maxlength]]" required$="[[required]]" disabled$="[[disabled]]">
 
-      <template is="x-if" if="[[errorMessage]]">
+      <template is="dom-if" if="[[errorMessage]]">
         <paper-input-error>[[parent.errorMessage]]</paper-input-error>
```
Comment (cdata) [inline paper-input.html]: `parent` is deprecated.

**Comment by notwaldorf** [comment]
Done! (minus the defaults bit, which I'm punting)

**Comment by morethanreal** [comment]
lgtm

**Pull request merged**